architecture-decision-records/autox/ODH-ADR-AX-0001-manage-code-duplication-automl-autorag.md

```
@@ -0,0 +1,216 @@
+**Architecture**: Primitives → features → pages composition hierarchy. Configure as Module Federation shared singleton.
+
+### Local Development
+
+* **BFF**: Go workspaces for seamless cross-package development
+* **UI**: npm workspaces for seamless cross-package development
+
+**Note**: `autox-core` can be maintained either in the current monorepo or in a separate repository. Both approaches are viable options.
+
+### Package Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)